README.md

```diff
@@ -32,7 +32,7 @@ Recent News
 There is a [new article (open access!)](https://www.sciencedirect.com/science/article/pii/S2665963820300403) in Software Impacts on 
 the t-digest, focussed particularly on this reference implementation.
  
-Lots has happened in t-digest lately. Most recently, with the help of people
+Lots have happened in t-digest lately. Most recently, with the help of people
```

Comment:
The standard idiom in American English is "lots has happened". Let's stick with that.